NuRadioMC/SignalGen/emitter.py

```diff
@@ -35,30 +44,42 @@ def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
         * idl1 & hvsp1 : these are the waveforms generated in KU lab and stored in hdf5 files
         * gaussian : represents a gaussian pulse where sigma is defined through the half width at half maximum
         * ARA02-calPulser : a new normalized voltage signal which depicts the original CalPulser shape used in ARA-02
+        * efield_idl1_spice: direct measurement of the efield from the idl1 pulser and its antenna as used in the SPICE
```

Comment:
please clarify in the comment that the original in-air measurement was corrected to in-ice emission with n=1.78 according to f = f_air / n (see appendix 1.4 of the NuRadioReco paper)https://link.springer.com/article/10.1140/epjc/s10052-019-6971-5#appendices
and double-check that this was the case.